app/Http/Controllers/ChannelController.php

```diff
@@ -127,8 +127,6 @@ public function destroy($id)
             return response()->json(['message' => 'Het kanaal werd niet gevonden.'], 400);
```

Comment:
Make message translatable

Reply:
will be done in issue  https://github.com/StadGent/laravel_site_opening-hours/issues/85